app-emulation/euae/euae-0.8.29.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="E-UAE Experimental Ubiquitous Amiga Emulator"
+DESCRIPTION="This is a version of UAE, the Ubiquitous Amiga Emulator, with an \
+emulation core largely based on WinUAE. It attempts to bring many of the great\
+ features of WinUAE to non-Windows platforms. This version now finally has a n\
+ame, E-UAE, since that's what everybody was calling it anyway. The 'E' can sta\
+nd for anything you fancy. Experimental, extreme, exciting, egalitarian, eggpl\
+ant, . . .
+Currently it will build and run (with a varying degree of supported features) \
+on Linux and other Unices, Mac OS X, BeOS, AmigaOS itself (either for 68k mach\
+ines or PPC machines with AmigaOS 4.0 and the AmigaOS clones MorphOS and AROS)\
+. OS X requires either LibSDL or an X server for graphics output, but native g\
+raphics are supported on AmigaOS and BeOS, although, at the moment, SDL is als\
+o preferred on BeOS since the native driver is incomplete.
+E-UAE is open-source software and is made available under the terms of the GNU\
+ GPL. E-UAE is based on the work of dozens of contributors including Bernd Sch\
+ midt (the original author and maintainer of UAE), Bernie Meyer (the author of\
+  the x86 JIT compiler), Toni Wilen (the current maintainer of WinUAE), and ma\
+  ny more.
+To make full use of E-UAE you will need access to an image of some version of \
+the Amiga Kickstart ROM (although UAE does include a ROM emulation which may w\
+ork with some old games). The Amiga Kickstart ROMs are copyrighted material an\
+d may not be freely distributed (so don't ask me for a copy). If you don't hav\
+e an Amiga from which you can legally make a ROM image, a selection of ROM ima\
+ges are available for purchase online from Cloanto - publishers of the Amiga F\
+orever distribution."
+HOMEPAGE="http://www.rcdrummond.net/uae/"
+COPYRIGHT="2004-2016 Richard Drummond"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.rcdrummond.net/uae/e-uae-0.8.29-WIP4/e-uae-0.8.29-WIP4.tar.bz2"
+CHECKSUM_SHA256="9828cb0fca783160bcb33900d0254f4bdaade1c9a41256e5f298e973d0c07456"
+SOURCE_DIR="e-uae-0.8.29-WIP4"
+
+ARCHITECTURES="x86_gcc2"
+
+PROVIDES="
+	euae = $portVersion
+	cmd:uae
+	cmd:readdisk
+	"
+REQUIRES="
+	haiku
+	lib:libbz2
+	lib:libGL
+	lib:libpng
+	lib:libSDL_1.2
+	lib:libsdl_sound
+	lib:libxml2
+	lib:libz
+	lib:libao
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libbz2
+	devel:libpng
+	devel:libsdl
+	devel:libsdl_sound
+	devel:libxml2
+	devel:libz
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:bison
+	cmd:flex
+	cmd:gcc
+	cmd:make
+	cmd:perl
+	cmd:pkg_config
+	cmd:libtoolize
+	"
+
+BUILD()
+{
+	libtoolize --force --copy --install
+	runConfigure ./configure --with-sdl --with-sdl-gfx --with-sdl-sound --enable-ui --enable-jit --disable-bsdsock
```

Comment:
Oops, I missed this one. Could you please split this line into two lines?
```
	runConfigure ./configure --with-sdl --with-sdl-gfx --with-sdl-sound \
		--enable-ui --enable-jit --disable-bsdsock
```
Reminder: we have one tab before the first half of the command, and two before the second half.

You can also split runConfigure command differently if you prefer, as long as each line does not exceed 80 chars and you use 1 tab for the first line and 2 for the following.
Of course, there are some exceptions: if you have, e.g. a sed command wich cannot be split then that's OK to have lines bigger than 80 chars.